spec/velveteen/worker_spec.rb

```diff
@@ -81,4 +136,24 @@ def perform
       )
     end
   end
+
+  class TestValidatesPayloadWorker < described_class
+    attr_accessor :test_headers
+    attr_reader :publish_routing_key, :publish_message, :headers
+
+    def initialize(message:, publish_message: nil, publish_routing_key: "velveteen.test.publish")
+      super(message: message)
+      @publish_routing_key = publish_routing_key
+      @publish_message = publish_message || message
```

Comment:
Both tests pass in a publish message. Can we remove this conditionals and the default `nil` value above?